Content.Client/Players/PlayTimeTracking/JobRequirementsManager.cs

```diff
@@ -177,4 +176,24 @@ public TimeSpan FetchOverallPlaytime()
 
         return _roles;
     }
+
+    public bool IsRoleBanned(IEnumerable<string> roles, out string? banReason, out DateTime? expirationTime)
+    {
+        banReason = null;
+        expirationTime = null;
+
+        foreach (var role in roles)
+        {
+            var roleBan = _roleBans.FirstOrDefault(ban => ban.Role == role);
+            if (roleBan != null)
+            {
+                banReason = roleBan.Reason;
+                expirationTime = roleBan.ExpirationTime;
+                return true;
+            }
+        }
+
+        return false;
+    }
```

Comment:
You should also really add a method to get job / antag bans as a helper in case we ever update the internals.

Reply:
Every time I see "Job:" or "Antag:" I contemplate if its worth doing a migration to have a Ban type in that table. 
Added the helpers.